zipline/src/hostMain/kotlin/app/cash/zipline/Zipline.kt

```diff
@@ -143,6 +143,9 @@ actual class Zipline private constructor(
     closed = true
 
     scope.cancel()
+    for (serviceName in endpoint.inboundServices.keys.toList()) {
```

Comment:
Why the copy? Does disconnect mutate? Can we use toTypedArray then if so? It's cheaper to create and cheaper to iterate. Plus a comment, please!

Reply:
Yep, disconnect mutates inboundServices.